### DISCUSSION TITLE
Problema na integracao com Checkout Bricks

### DISCUSSION THREAD

**Discussion by wilsntn** [mercadopago/sdk-react#94] [Q&A] [answered]
segui esse [https://github.com/mercadopago/sdk-react/wiki/Using-Bricks-with-Next.js](url) da wiki mas ainda estou com problema para carregar o componente está dando um erro de uma requisicao que e feita pelo componente 
![image](https://github.com/mercadopago/sdk-react/assets/91102332/720ede8c-7e8c-4684-8c07-925b52f48bd8)
e o componente nao aparece na tela. Estou utilizando nextJS 14 alguem conseguiu integrar esse meio de pagamento com essa versão?

**Comment by francineemilia** [comment]
> segui esse [https://github.com/mercadopago/sdk-react/wiki/Using-Bricks-with-Next.js](url) da wiki mas ainda estou com problema para carregar o componente está dando um erro de uma requisicao que e feita pelo componente ![image](https://private-user-images.githubusercontent.com/91102332/313504867-720ede8c-7e8c-4684-8c07-925b52f48bd8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3NzEzMDcsIm5iZiI6MTcxMDc3MTAwNywicGF0aCI6Ii85MTEwMjMzMi8zMTM1MDQ4NjctNzIwZWRlOGMtN2U4Yy00Njg0LThjMDctOTI1YjUyZjQ4YmQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE4VDE0MTAwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhNmQzYmRmMzIxMzA4NGIwNmRjYjBkMGFhMjgzZGFmMWFhMjI1NmE4OWJjMTJjMzQ0ZDE1OGJlYWIwOGIyYTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LRyP15riQ3j8YwCHK11nPtvdeNGED6PbbAZLp0xvfq4) e o componente nao aparece na tela. Estou utilizando nextJS 14 alguem conseguiu integrar esse meio de pagamento com essa versão?

@wilsntn qual o componente que você está tentando utilizar? Pode enviar um exemplo de seu código nesta thread?

**Reply by wilsntn** [reply, marked as answer]
Descobri o problema, estava passando o token ao invez do usertoken!
